components/cards.css

```diff
@@ -0,0 +1,564 @@
+@import url("/documentation/doc.css");
+.cards {
+  display: flex;
+  flex-direction: column;
+}
+.cards-shape {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 2rem 0rem 2rem 0rem;
```

Comment:
For this you can use shorthand , `margin : top&botton left&right`. In this case ,
 it would like `margin : 2rem 0rem`;

Reply:
hey, @devananth changes have been done according to shorthand property. will use everywhere. Thanks for the feedback.